spec/facade/food_facade_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade do 
+  it "gets data for food by ingredient", :vcr do 
+    foods = FoodFacade.create_foods('sweet potatoes')
+
+    expect(foods[0]).to be_a(Food)
```

Comment:
You can also make assertions about how many Food items are in the array (10). that the top level data structure is an array, etc. 

We would also want you to add an assertion about the presence of the total hits in your returned data